apps/right-to-rent-check/views/tenant-another.html

```diff
@@ -5,37 +5,47 @@
       {{#renderField}}{{/renderField}}
     {{/fields}}
     {{#input-submit}}continue{{/input-submit}}
-    <h2>Tenants details</h2>
+    {{#hasTenants}}
+      <h2>{{#t}}pages.tenant-another.tables.title{{/t}}</h2>
+    {{/hasTenants}}
     {{#tenants}}
       <table class="tenant-details">
         <tbody>
           <tr>
             <th>{{#t}}pages.tenant-another.tables.headers.name{{/t}}</th>
             <td>{{tenant-name}}</td>
+            <td><a href="/tenant-another/edit/{{tenant-uuid}}?field=tenant-name">{{#t}}buttons.change{{/t}}</td>
```

Comment:
change-tenant-name in the class it'll make your tests less fragile and more readable.

rinse and repeat for each one